provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/client/VlerClient.java

```diff
@@ -0,0 +1,45 @@
+package gov.va.api.health.providerdirectory.service.client;
+
+import gov.va.api.health.providerdirectory.service.AddressResponse;
+
+/** This is the abstraction for communicating with PPMS through their Restful API. */
+public interface VlerClient {
+
+  AddressResponse endpointByAddress(String address);
+
+  /** A request was malformed, such as missing required search parameters. */
+  public static final class BadRequest extends VlerException {
+
+    public BadRequest(String message, Throwable cause) {
```

Comment:
Are these all the same exceptions that were defined for PPMS? Seems like we should consolidate these into a shared `Exceptions` class